fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTableStatsStmt.java

```diff
@@ -75,8 +75,10 @@ public class ShowTableStatsStmt extends ShowStmt implements NotFallbackInParser
             new ImmutableList.Builder<String>()
             .add("table_name")
             .add("index_name")
-            .add("row_count")
+            .add("analyze_row_count")
             .add("update_time")
+            .add("report_row_count")
+            .add("report_row_count_for_nereids")
```

Comment:
report_row_count_for_nereids: what is the meaning of this row count?

Reply:
report_row_count_for_nereids return -1 when part of the tablets are reported, report_row_count always return the current value of reported row count, no matter it's partially reported or not.

Reply:
in this case, report_row_count should be "-1"